git-extra/PKGBUILD

```diff
@@ -5,7 +5,7 @@ pkgbase="mingw-w64-${_realname}"
 pkgname=($_realname
         "${MINGW_PACKAGE_PREFIX}-${_realname}")
 _ver_base=1.1
-pkgver=1.1.641.031e03baf
+pkgver=1.1.643.43da6798e
```

Comment:
The commit message says "update checksums", but no checksums where updated.

Reply:
Right, I just imitated the last commits of this kind, they were all committed directly by our automation, using that commit message.

Reply:
I hence fixed this, it now reads:

```
    git-extra: commit updates made by `sdk build`

    The way the `git-extra` package is designed, it updates the version as
    well as the `git-extra.install` file as part of running `makepkg-mingw`.

    These are the updates after the parent commit changed
    `git-extra.install.in` to introduce a hack to allow gnupg/gnutls to find
    `libunistring`'s DLL again.

    Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
```